tests/acc_test.go

```diff
@@ -1182,6 +1182,37 @@ func TestE2eYAML(t *testing.T) {
 	}
 }
 
+func TestEndToEndUsingModuleWithDashes(t *testing.T) {
+	localProviderBinPath := ensureCompiledProvider(t)
+	localPath := opttest.LocalProviderPath("terraform-module", filepath.Dir(localProviderBinPath))
+	modulePath, err := filepath.Abs(filepath.Join("testdata", "modules", "dashed-module-fields"))
+	assert.NoError(t, err, "failed to get absolute path for dashed module")
+	// TODO[pulumi/pulumi-terraform-module#426] - test local modules with Go
+	for _, runtime := range []string{"yaml", "ts", "python", "dotnet"} {
+		t.Run(runtime, func(t *testing.T) {
+			testProgram, err := filepath.Abs(filepath.Join("testdata", "programs", runtime, "dashed_module"))
+			require.NoError(t, err, "failed to get absolute path for dashed_module program")
+			tfFilesOutputDir := filepath.Join(testProgram, "tf_files")
```

Comment:
what's the purpose of this directory for tf output files? Is it necessary for the tests to run? it doesn't look like we check it.